easybuild/framework/easyconfig/format/yeb.py

```diff
@@ -63,9 +63,26 @@ def yaml_join(loader, node):
     seq = loader.construct_sequence(node)
     return ''.join([str(i) for i in seq])
 
-# register the tag handler
-yaml.add_constructor('!join', yaml_join)
+def yaml_versionmajorminor(loader, node):
+    """
+    defines custom YAML function to get major version.
+    @param loader: the YAML Loader
+    @param node: the YAML (sequence) node - for now, ideally this would be a scalar
+    """
+    # this should be fixed
+    if isinstance(node, yaml.nodes.ScalarNode):
+        pyver = loader.construct_scalar(node)
+    elif isinstance(node, yaml.nodes.SequenceNode):
+        pyver = loader.construct_sequence(node)
+        pyver = pyver[0]
```

Comment:
this needs a check: if the `pyver` sequence has more than one element, we're in trouble here (that is, we shouldn't be simply ignored all the other elements, we should report it as a problem/syntax error)